pkg/prow/prow.go

```diff
@@ -3,6 +3,7 @@ package prow
 import (
 	"bytes"
 	"fmt"
+	"github.com/openshift/ci-chat-bot/pkg/prow/apiv1"
```

Comment:
What is this?

Reply:
Probably easier for someone's overloaded brain not to have this called Prow API since it's not, but I don't care too much